Automattic/jetpack
# Boost: Fix wp-config lookup

## Proposed changes:
It would potentially break the site if wp-config.php file was not located in the `ABSPATH` but on the parent directory. The problem is now fixed and it will try to locate the correct location of wp-config.


### Other information:


## Jetpack product discussion
No

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Create a JN site or any WP site where wp-config is located in the parent directory of webroot.
- Remove `advanced-cache.php` file if that's already there.
- Install boost.
- Try setting up cache.
- Make sure it modified the correct wp-config.php file.
